### ISSUE TITLE
End of waveform is couple milliseconds before the end of video's last frame

### ISSUE THREAD

**Issue by rRobis** [SubtitleEdit/subtitleedit#7859]
I noticed, that when I go to the end of the video (just play until it stops) and set the end of the last subtitle there, it doesn't show in the last frame of the video. 

I have to add couple milliseconds to the last subtitle elements for them to show, otherwise you can see that the subtitle elements disappear from the video in the last frame. 

This is annoying in doing shorts for example, where i have subtitle background and looping, and have to adjust the length every time for all the elements.

**Comment by LeonCheung** [comment]
Guess what, a quick and dirty way to solve this is by simply hitting the button/shortcut twice... sometimes😂

**Comment by niksedk** [comment]
Would added a tiny bit silence to the end of the waveform help?

https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.5/SubtitleEditBeta.zip

**Comment by rRobis** [comment]
It seems that it could be caused by the way i make the file with ffmpeg.

I checked another file, that i rendered in Davinci resolve, and when the video stops and i set end of subtitle there, then i can see the subtitle on last frame (working as expected).

But when using file that i rendered using ffmpeg (and added some extra overlay with it), i have to extend it, to be visible on the last frame.. So maybe it's something to do with the way that i create these specific files.

**Comment by niksedk** [comment]
OK, unless anyone think it can hurt, SE will add few milliseconds to the waveform (as in latest beta)

**Comment by rRobis** [comment]
Actually, i just found out the source of the problem. ChatGPT gave me an idea 😄 

I looked up with `ffprobe -i subtitles.mp4 -show_streams` 

and it turned out, that 

video and audio streams were a little bit different length

duration=59.680000
duration=59.605000

I added -shortest to the ffmpeg and now it's working as expected. I guess it works without any noticeable problems in this case, because it's not a significant difference.
